app/models/tenant.rb

```diff
@@ -317,6 +318,10 @@ def create_miq_product_features_for_tenant_nodes
     MiqProductFeature.seed_single_tenant_miq_product_features(self)
   end
 
+  def update_miq_product_features_for_tenant_nodes
+    MiqProductFeature.invalidate_caches_queue
```

Comment:
Do we want to be more selective about which changes need to update the product features?

Reply:
I had asked Libor the same question.

It would mean that we know in this class exactly what the miq product feature cache is holding. and if the business logic or display logic over there changes.

I deemed that we don't make many tenant changes other than name adds and removes, so the blanket change was the safest bet and not overly cache invalidation happy.

If you have another interpretation, please share

Reply:
I'll put in after_create per your other request

Reply:
So my argument over there was that the logic doesn't belong in the UI but did on the model, so at least all of the "information" would be self contained... But I agree that you'd have to have model logic to know this relationship anyway and then remember to update both sides, and that's not ideal.